code/API_definitions/Get DeviceLocation.yaml

```
@@ -0,0 +1,344 @@
+openapi: 3.0.3
+info:
+  title: Device location retrieval API
+  description: Service Enabling Network Function API for location retrieval from a device identifier.
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+  - three_legged:
+    - location-retrieval-read
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location/v0
+        description: Base path for the device location API
+paths:
+  /retrieve-location:

```

```suggestion
  /retrieve:
```
Agreed after #84, to consolidate naming in both APIs